README.md

```diff
@@ -1,5 +1,167 @@
-# Reconstruction tools
+# Reconstruction pipeline
 
-Tools for building full brain network models from standard structural MR scans.
+This pipeline is offering a solution to build full brain network models starting from standard structural MR scans.
+It is used to preprocess the MR scans in order to get actual files that are compatible with TVB. The result can be later uploaded in TVB. 
+
+The mandatory inputs are DWI and T1 scans. Optionally, CT scans can be given as input, if sensors preprocessing is needed.
+
+We are using the Pegasus WMS in order to connect and automatize the pipeline steps.
+
+## Folder structure
+- data
+	<br> Here is where we keep some example data files. Some of them are intermediate files generated during pipeline run. There is also a minimal set of files that defines a TVB head. These files are currently used only for tests.  
+- docs
+	<br> This folder holds some visual hints of the dependencies between the pipeline steps. For example, there is an overview diagram representing the pipeline stages. These stages are defined by scripts that can be found inside the bin folder. On the other hand, there is also an example graph diagram which is displaying the more detailed steps. This kind of diagram is automatically generated at each pipeline run.  
```

Comment:
these scripts are no longer maintained, nor used directly in the pipeline run. They belong under "docs" because now are only for documentation purposes used, or for some extreme partial debugging.